Core/Security/VOMS.py

```diff
@@ -31,12 +31,6 @@ def getVOMSAttributes( self, proxy, switch = "all" ):
 
     #Get a list of known VOMS attributes
     validVOMSAttrs = []
-    result = gConfig.getOptions( "/Registry/VOMS/Mapping" )
-    if result[ 'OK' ]:
-      for group in result[ 'Value' ]:
-        vA = gConfig.getValue( "/Registry/VOMS/Mapping/%s" % group, "" )
-        if vA and vA not in validVOMSAttrs:
-          validVOMSAttrs.append( vA )
```

Comment:
This is an obsoleted section which is replaced by VO section. Anyway the valid attributes are all collected from the group definitions afterwards.